src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -355,4 +358,64 @@ public without sharing class UTIL_CustomSettingsFacade {
             }
         }
     }
+
+    /*******************************************************************************************************
+    * @description Gets if Course Connections is enabled
+    * @return Boolean
+    */
+    public static Boolean courseConnectionsEnabled() {
+        if (!Test.isRunningTest()) {
+            return (
+                getSettings().Enable_Course_Connections__c &&
+                getSettings().Default_Enrolled_Student_RecordType_Id__c.length() >= 15 &&
+                getSettings().Default_Faculty_RecordType_Id__c.length() >= 15 &&
+                getSettings().Default_Faculty_RecordType_Id__c != getSettings().Default_Enrolled_Student_RecordType_Id__c
+            );
+        }
+        return getSettings().Enable_Course_Connections__c;
+    }
+
+    /*******************************************************************************************************
+     * @description Returns the ID of the Faculty Course Connection record type, if it exists.
+     * @return String The ID of the Faculty Course Connection record type as a String.
+     */
+    public static String getFacultyConnectionRecTypeID() {
+        String recTypeId = getSettings().Default_Faculty_RecordType_Id__c;
+        if(Test.isRunningTest() && String.isBlank(recTypeId)) {
+            String defaultRecTypeId = UTIL_Describe.getRecTypesMapByDevName('Course_Enrollment__c').get('Faculty');
```

Comment:
You could probably simplify the code a little by not declaring a second variable, defaultRecTypeId, and just use recTypeId.